lib/rspec/rails/example/system_example_group.rb

```diff
@@ -100,6 +100,10 @@ def driven_by(driver, **driver_options, &blk)
           @driver = ::ActionDispatch::SystemTestCase.driven_by(driver, **driver_options, &blk).tap(&:use)
         end
 
+        def url_options
```

Comment:
@JonRowe this method is never invoked by `DISABLE_SPRING=1 bundle exec rspec spec/system/websites_spec.rb` on the example app in #2327